### DISCUSSION TITLE
Ability to use custom rules as well as default

### DISCUSSION THREAD

**Discussion by Roxedus** [Checkmarx/kics#6819] [Ideas] [answered]
While the documentation lays out how to do this, by using `secrets-regexes-path` and `include-queries`, the inclusion `include-queries` discards the default (all?) queries. 
A solution like a shortcut for all queries for `include-queries`, like star(*) would be appreciated.

**Comment by gabriel-cx** [comment, marked as answer]
Hi @Roxedus,

Thanks for your inputs!
Indeed, `include-queries` makes KICS run the mentioned queries and discard the others.
I do believe, however, you do not need the use of this flag to achieve what you want.

It's possible for you to provide more context about how do you use KICS? (do you use the [KICS version official docker image](https://hub.docker.com/r/checkmarx/kics/tags)? do you use the repository master content and create your own KICS docker image? Kindly provide also a simple example of your use case: _actual behavior_ vs _expected behavior_).
With that information we can provide you better solutions.
